Mesh_3/include/CGAL/Mesh_3/tet_soup_to_c3t3.h

```diff
@@ -1,6 +1,9 @@
 #ifndef CGAL_MESH_3_TET_SOUP_TO_C3T3_H
 #define CGAL_MESH_3_TET_SOUP_TO_C3T3_H
 
+#include <CGAL/license/Mesh_3.h>
+
```

Comment:
In addition this new file should also have the usual copyright+license preamble, like any other header file include `include/CGAL/`.